docs/usage/filters.md

```diff
@@ -0,0 +1,291 @@
+# Filter knowledge
+
+In OpenCTI, you can filter data to target or view entities that have particular attributes.
+
+## Filters usages
+Filters are used in different locations in the platform.
+- in entities lists: filters restrict the displayed entities to the ones respecting the chosen filtering. If an export or a background task is generated, only the filtered data will be taken into account.
+- in investigations and knowledge graphs: filters restrict the displayed entities to the ones respecting the chosen filtering.
+- in workspaces: filters enable to create graphs with only the wanted entities.
+- in feeds, taxii collections, triggers, streams, playbooks: filters restrict the data to manage or the events to listen to.
+
+There are two types of filters:
+- dynamic filters: They are not stored in the database, they enable to filter view in the UI. Exemples: filters in entities list, investigations, knowledge graphs;
+- stored filters: They are attributes of an entity, they are stored as they other attributes. Exemples: filters of workspaces, feeds, taxii collections, triggers, streams, playbooks.
+
+## Filters format (since 5.12)
```

Comment:
We could split in 2 sections : UI and API usages. 
This section here is the API.

Reply:
right!